### PR TITLE
DAC6-1418 | Changed routing for assistants taking them to MDR frontend

### PR DESCRIPTION
**Pull request by matthewharris1**

### CODE REVIEW

**Review by iservport — APPROVED** [review]
LGTM

File: app/controllers/actions/IdentifierAction.scala [hunk 1]
```diff
@@ -70,6 +70,8 @@ class AuthenticatedIdentifierActionWithRegime @Inject() (
       .retrieve(Retrievals.internalId and Retrievals.allEnrolments and affinityGroup and credentialRole) {
         case _ ~ _ ~ Some(Agent) ~ _ =>
           Future.successful(Redirect(controllers.routes.UnauthorisedAgentController.onPageLoad(regime)))
+        case _ ~ enrolments ~ _ ~ Some(Assistant) if enrolments.enrolments.exists(_.key == "HMRC-MDR-ORG") =>
```
Comment (ashwinirudrappa) [inline app/controllers/actions/IdentifierAction.scala:73]: @matthewharris1 Let the config choose the regime from application conf
keys {
  enrolmentKey.mdr = "HMRC-MDR-ORG"
}
```suggestion
        case _ ~ enrolments ~ _ ~ Some(Assistant) if enrolments.enrolments.exists(_.key == config.enrolmentKey(regime.toString)) =>
```

**Review by ashwinirudrappa — COMMENTED** [review]
LGTM, rather than hardcoding the enrolment key lets get it from conf

File: app/controllers/actions/IdentifierAction.scala (hunk 1 above)
Comment (matthewharris1) [reply app/controllers/actions/IdentifierAction.scala]: Good idea fixed!

**Review by ashwinirudrappa — APPROVED** [review]

**Pull request merged**